containers/kubernetes/api-cli/managing-storage.mdx

```diff
@@ -18,175 +18,89 @@ categories:
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have [created](/containers/kubernetes/how-to/create-cluster/) a Kubernetes cluster running on Scaleway Instances (v1.21+)
-  - You have a Scaleway Project or Organization ID, [Access and Secret key](/identity-and-access-management/iam/how-to/create-api-keys/)
 </Message>
 
+The Scaleway Block Volume CSI driver is responsible for the management of Scaleway Block Volumes within Kubernetes. Pre-installed on Kubernetes Kapsule and Kosmos clusters, this driver allows you to control Block Volumes directly from your orchestration environment. The plugin resides in the `kube-system` namespace within the pods.
 
-The [Scaleway Block Volume](https://www.scaleway.com/en/block-storage/) Container Storage Interface (CSI) driver is an implementation of the [CSI interface](https://github.com/container-storage-interface/spec/blob/master/spec.md) to provide a way to manage Scaleway Block Volumes through a container orchestration system, like Kubernetes. It is installed by default on every Kubernetes Kapsule and Kosmos cluster.
+<Message type="tip">
+    Refer to our video tutorial [Getting Started with Kubernetes Part 4 - Storage](/containers/kubernetes/videos/) to view a visual presentation and step-by-step guidance of how to manage Block Storage volumes on Kubernetes with the Scaleway CSI.
+</Message>
+
+## Verification of CSI driver status
+
+To verify if the driver is running, use the following command:
 
+```bash
+kubectl get csidriver
+```
 
-## Features
+The output of this command provides a quick status update on the CSI plugin within your Kubernetes cluster. For the latest features and enhancements, consider upgrading to [release 0.3](https://github.com/scaleway/scaleway-csi/tree/release-0.3#block-storage-low-latency), which supports **[Block Storage low latency](/storage/block/quickstart/)** volumes.
 
-Following is a list of functionalities implemented by the Scaleway CSI driver.
+To identify your current CSI release version, navigate to the [Cockpit interface](/observability/cockpit/how-to/access-grafana-and-managed-dashboards/), specifically the **Kubernetes Cluster - Overview** dashboard.
 
-### Block device resizing
+<Message type="note">
+   * Block volumes and snapshots management have transitioned to the [Scaleway Block Storage API](https://www.scaleway.com/en/developers/api/block/). This means, you can view your block volumes in the [Block Storage](https://console.scaleway.com/block-storage/volumes) section of the Scaleway console.
+   * Anticipate the availability of the **boot on block** feature for low-latency Block Storage volumes in Q2.
```

Comment:
It seems that this mention is obsolete/not necessary.

Reply:
Fixed